app/views/steps/check/results/shared/_meaning_cautions.en.html.erb

```diff
@@ -1,34 +1,33 @@
 <% if caution_type.youth? %>
-
   <h2 class="govuk-heading-m">
     If a basic or standard criminal record check is requested
   </h2>
 
   <p class="govuk-body">
-    You don’t need to tell people about this caution, reprimand or warning once it is spent, as you were under 18 when
-    you received it. It won’t appear on a
-    <a class="govuk-link" rel="external" target="_blank" href="https://www.gov.uk/guidance/telling-people-about-your-criminal-record#basic-dbs">basic
-      or standard criminal record check</a>.
+    You don’t need to tell people about this caution, reprimand or warning once
+    it is spent, as you were under 18 when you received it.
+    It won’t appear on a basic or standard criminal record check.
   </p>
 
   <h2 class="govuk-heading-m">
     If an enhanced criminal record check is requested
   </h2>
 
   <p class="govuk-body">
-    You probably won’t need to tell people about this caution, reprimand or warning once it is spent, as you were under
-    18 when you received it.
+    You probably won’t need to tell people about this caution, reprimand or warning once it is spent, as you were under 18 when you received it.
   </p>
 
   <p class="govuk-body">
-    However, some kinds of jobs may need you to give details of all cautions, reprimands and warnings even when they are
-    spent. Read more about
-    <a class="govuk-link" rel="external" target="_blank" href="https://www.gov.uk/guidance/telling-people-about-your-criminal-record#enhanced-dbs">enhanced
-      criminal record checks</a>.
+    However, some kinds of jobs may need you to give details of all cautions, reprimands and warnings even when they are spent.
   </p>
 
+  <p class="govuk-body">
+    See more about DBS checks and
```

Comment:
Be careful, in the past "see" was raised as a not appropriate accessible verb (some people can't see). 
That's why in PR #453 I used **read** instead.